cli/Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
 name = "localtunnel"
-version = "0.1.1"
-edition = "2021"
+version.workspace = true
```

Comment:
Different crates should maintain its version separately, otherwise it will break the [semver](https://semver.org/).

Reply:
You meant to say server / client / cli will follow different versions.

Reply:
Yes, that's why it's split into different crates.